packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -81,18 +95,32 @@ describe("pluggable-widgets-tools commands", () => {
                 process.stderr.write(`[${widgetName}] Testing unit tests...\n`);
                 await execFailedAsync("npm test", workDir);
                 await execAsync("npm test -- -u", workDir);
+                expect(existsSync(join(workDir, `/dist/coverage/clover.xml`))).toBe(true);
             });
         } else {
             it("tests succeed", async () => {
                 process.stderr.write(`[${widgetName}] Testing unit tests...\n`);
                 await execAsync("npm test", workDir);
+                expect(existsSync(join(workDir, `/dist/coverage/clover.xml`))).toBe(true);
             });
         }
 
         if (!LIMIT_TESTS) {
             it.each(["build", "test:unit", "release"])("'%s' command succeeds", async cmd => {
                 process.stderr.write(`[${widgetName}] Testing '${cmd}' command...\n`);
                 await execAsync(`npm run ${cmd}`, workDir);
+                if (cmd === "build") {
```

Comment:
Should we have a similar test for `release`? And if so, maybe we should split unit tests to a separate `it`, or even split all 3 of them to avoid such kind of `if`s

Reply:
Added the same check for `release` command. Also splitted the `it.each` into 3 seperate `it`'s.